src/models/serw13/repairs.md

```diff
@@ -0,0 +1,243 @@
+
+7. Carefully replace the heatsink.
+8. Replace the fan and heatsink screws, starting with #1, then #2, and so on until #13.
+9. Plug the white fan connectors back into the motherboard.
+
+## Replacing the wireless card:
+
+Your Serval WS's WiFi and Bluetooth are both handled by the same module. It is a standard M.2 2230 slot with PCIe and USB interfaces (A Key).
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 10 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>  
+
+### Steps to replace the WiFi/Bluetooth module:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. If there is an M.2 SSD in the slot closest to the battery, follow the steps above to [remove the M.2 SSD](#replacing-an-m2nvme-ssd).
+    - The other SSD can optionally be removed to make working with the wireless card easier.
+3. Locate the wireless module. Remove any clear tape that is securing the wires.
+
+![Wireless card](./img/wireless-card.webp)
+
+4. Gently remove the two antennas (highlighted red above) by pulling them up and away from the wireless card.
+5. Remove the retaining screw opposite the M.2 slot, highlighted yellow above.
+6. Slide the card out of the M.2 slot, parallel with the motherboard.
+7. Insert the new wireless card into the M.2 slot.
+8. Replace the retaining screw.
+9. Attach the two antennas by aligning the circular fittings and pressing onto the wireless card. The connectors will snap into place. _Use caution when attaching the connectors; the pins can bend, break, or snap._
+10. Replace the M.2 SSD, bottom cover, and battery.
+
+## Replacing the CMOS battery:
+
+The CMOS battery supplies power to the system's CMOS chip. UEFI settings and the comptuer's hardware clock are stored on the CMOS. If your system doesn't boot, you can reset the CMOS to force a low-level hardware reset. If your clock is constantly resetting, it's likely your CMOS battery needs to be replaced.
+
+**Tools required:** Cross-head (Phillips) screwdriver    
+**Time estimate:** 10 minutes    
+**Difficulty:** Easy <span style="color:green;">●</span>
+
+### Steps to replace the CMOS battery:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover) and [outermost SSD](#replacing-an-m2nvme-ssd).
+2. Remove the [innermost SSD](#replacing-an-m2nvme-ssd) and [remove the wireless card](#replacing-the-wireless-card), then pull the battery away from the adhesive holding it to the case.
+    - If you are not replacing the CMOS battery, then this step is optional.
+
+![CMOS battery](./img/cmos-battery.webp)
+
+3. Unplug the white connector that connects the CMOS battery to the motherboard.
+4. To clear the CMOS, open the lid of the machine and hold down the power button for at least 15 seconds to discharge any residual energy in the system.
+5. Re-connect the CMOS battery.
+6. Replace the bottom panel and main battery.
```

Comment:
Replacing the main battery is mentioned, but unplugging or removing it isn't mentioned prior. It doesn't look like fully removing the main battery is necessary, but unplugging it before trying to clear CMOS should probably be mentioned.

Reply:
cb88769edd999a58d754f7b9586cf00b8da76e9e should make a little more sense.